modules/tinymce/src/themes/silver/main/ts/ui/dialog/Collection.ts

```diff
@@ -135,6 +136,33 @@ export const renderCollection = (
       }),
       ReadOnly.receivingConfig(),
       Replacing.config({ }),
+      Tooltipping.config(
+        {
+          ...providersBackstage.tooltips.getConfig({
+            tooltipText: ''
+          }),
+          mode: 'children-keyboard-focus',
+          onShow: (comp) => {
```

Comment:
Is there any conflict between this `comp` and the above `comp` parameter?.

Reply:
There shouldn't be any conflict at all 👍